### PR TITLE
Add Sidebar hidden/visible Classes 

### PR DESCRIPTION
**Pull request by mlmoravek**
Implements #435

## Proposed Changes
adds hiddenClass and visibleClass to Sidebar component

### CODE REVIEW

**Comment by jtommy** [comment]
@mlmoravek thanks, can you add the classes to inspectors too?

**Comment by mlmoravek** [comment]
> @mlmoravek thanks, can you add the classes to inspectors too?

@jtommy hope i did it right :)

File: packages/oruga/src/components/sidebar/Inspector.vue [hunk 1]
```diff
@@ -174,6 +174,20 @@ export default {
...
+                    description : 'Class of the sidebar when sidebar is visible',
+                    action: (cmp) => {
+                        this.position = 'static';
+                    }
+                },
+                {
+                    class: "hiddenClass",
+                    description : 'Class of the sidebar  when sidebar is hidden',
+                    action: (cmp) => {
+                        this.position = 'static';
```
Comment (jtommy) [inline packages/oruga/src/components/sidebar/Inspector.vue:188]: Why this?

**Review by jtommy — CHANGES_REQUESTED** [review]

File: packages/oruga/src/components/sidebar/Inspector.vue (hunk 1 above)
Comment (mlmoravek) [reply packages/oruga/src/components/sidebar/Inspector.vue]: copy paste leftovers. I removed the action for the hidden class.

File: packages/docs-next/.vitepress/theme/examples/sidebar/inspector.vue [hunk 2]
```diff
@@ -177,6 +177,17 @@ export default {
             cmp.data.variant = 'warning';
           }
         },
+        {
+            class: "visibleClass",
+            description : 'Class of the sidebar when sidebar is visible',
+            action: () => {
+                this.position = 'static';
```
Comment (jtommy) [inline packages/docs-next/.vitepress/theme/examples/sidebar/inspector.vue:184]: Sane thing here

File: packages/oruga/src/components/sidebar/Inspector.vue [hunk 3]
```diff
@@ -174,6 +174,17 @@ export default {
                         cmp.data.variant = 'warning';
                     }
                 },
+                {
+                    class: "visibleClass",
+                    description : 'Class of the sidebar when sidebar is visible',
+                    action: () => {
+                        this.position = 'static';
```
Comment (jtommy) [inline packages/oruga/src/components/sidebar/Inspector.vue:181]: Same thing here

**Review by jtommy — CHANGES_REQUESTED** [review]

**Review by jtommy — APPROVED** [review]

**Pull request merged**